components/docs-chef-io/content/habitat/get_started.md

```diff
@@ -0,0 +1,257 @@
+
+Add your package to your Builder origin and then connect that package to your clone of the sample-node-app repository. Any changes made to your clone of the sample-node-app will trigger the Chef Habitat Builder to rebuild the package.
+
+1. If you haven't already done so, commit the changes you made earlier to the `plan.sh` file into the default branch of your clone of the sample-node-app repository, and push those changes up to your fork on GitHub.
+
+   The `pkg_origin` setting in the `plan.sh` file should be set to your origin.
+
+1. Log in to your account in [Chef Habitat Builder](https://bldr.habitat.sh) and select your origin.
+1. Create a new package by selecting **Create Package**.
+
+   The name of the package you create must match the package name defined by the `pkg_name` setting in the `plan.sh` file.
+
+1. Connect the plan file to the new package by selecting **Connect a plan file** and then **for Linux (kernel version 3.2 or later)**.
+1. Select your GitHub organization or user account under **Organization**, and then select **sample-node-app** under **Repository**.
+1. Select **Save Connection** at the bottom of the page.
+
+1. Once the plan is connected, select **Latest** towards the top of the page and build the package by selecting **Build latest version** on the right side of the page.
+
+   {{< figure src="/images/habitat/builder-build-latest-version.png" alt="Build latest version button." width=250 >}}
+
+   Select **View the output** and then the **View build job** button ({{< svg file="/static/images/habitat/builder-view-build-job-icon.svg" >}}) to watch the service building in Habitat Builder.
+
+1. After Habitat Builder finishes building the service, select the **Latest** tab, and then select **Promote** to promote the service to the stable channel.
+
+## Run the Service From Habitat Builder in the Habitat Studio
+
+Now that the Node.js app is under management by the Chef Habitat Builder, you can update the service from Habitat Builder.
+
+1. From a local terminal, enter the Habitat Studio:
+
+   ```bash
+   hab studio enter
+   ```
+
+1. Verify that a service is not running in the Studio:
+
+   ```bash
+   hab svc status
+   ```
+
+1. Load the service into the Studio using the at-once update strategy.
+
+   The [at-once update strategy]({{< relref "/habitat/service_group_updates#at-once-strategy" >}}) will pull down the package from Habitat Builder whenever it detects that a new version has been released.
+
+   ```bash
+   hab svc load ORIGIN_NAME/sample-node-app --strategy at-once
+   ```
+
+1. Now monitor the Habitat Supervisor to watch it update when a new version of the package is created.
+
+   On macOS or Linux, run:
+
+   ```bash
+   sup-log
+   ```
+
+   On Windows, run:
+
+   ```bash
+   Get-SupervisorLog
```

Comment:
They will not want to enter a windows studio for the sample node app since there is no plan.ps1.

Reply:
Does the sample-node-app tutorial make sense for Windows, or is there a powershell version of the sample-node-app?
It looks like it would take a bit of work for a user to convert this plan to a powershell script. More than I want to describe in a tutorial.

Reply:
Yeah it's probably best to stick to linux here.